docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,397 @@
+       STATUS: deployed
+       REVISION: 1
+       TEST SUITE: None
+       NOTES:
+       Get the application URL by running these commands:
+         export POD_NAME=$(kubectl get pods --namespace default -l "app=trino,release=my-trino-cluster,component=coordinator" -o jsonpath="{.items[0].metadata.name}")
+         echo "Visit http://127.0.0.1:8080 to use your application"
+         kubectl port-forward $POD_NAME 8080:8080
+
+   This output depends on your ``values.yaml`` configuration and cluster name.
+   For example, the port ``8080`` is set by the ``.service.port`` in the
+   ``values.yaml``.
+
+#. Run the following command to check that all pods, deployments, and services are
+   running properly.
+
+   .. code-block:: text
+
+      kubectl get all
+
+   You should expect to see output that shows running pods, deployments, and
+   replica sets in the ready state.
+
+   .. code-block:: text
+
+       NAME                                               READY   STATUS    RESTARTS   AGE
+       pod/my-trino-cluster-coordinator-bfb74c98d-rnrxd   1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-hvl8n       1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-tcqgb       1/1     Running   0          161m
+
+       NAME                       TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)    AGE
+       service/my-trino-cluster   ClusterIP   10.96.25.35   <none>        8080/TCP   161m
+
+       NAME                                           READY   UP-TO-DATE   AVAILABLE   AGE
+       deployment.apps/my-trino-cluster-coordinator   1/1     1            1           161m
+       deployment.apps/my-trino-cluster-worker        2/2     2            2           161m
+
+       NAME                                                     DESIRED   CURRENT   READY   AGE
+       replicaset.apps/my-trino-cluster-coordinator-bfb74c98d   1         1         1       161m
+       replicaset.apps/my-trino-cluster-worker-76f6bf54d6       2         2         2       161m
+
+   * The top three lines are pods. These include the actual Trino containers.
+
+   * The next line is the service. This is a mechanism by which Kubernetes makes
+     internal services like the one running on the coordinator accessible
+     within the cluster by using a name instead of IPs.
+
+   * The next two lines are the deployments. Deployments contain logic around
+     how upgrades are performed, and manage pods through replica sets.
+     In the case of ``my-trino-cluster-worker`` deployment, there are multiple
+     pods. Notice that the ``my-trino-cluster-coordinator`` always has one pod.
+
+   * The final two lines are the replica sets. This defines how many instances
+     of a given pod should be available during a ready state. This is the way
+     to scale out the deployments.
+
+#. If all pods, deployments, and replica sets are running and in the ready
+   state, Trino has been successfully deployed.
```

Comment:
"ready state" doesn't map directly to anything in the output. What else can we say? I think key is to see if pods have 1/1 in READY column (everything else should be good if that is good ideally).